dvc/remote/azure.py

```diff
@@ -136,9 +133,20 @@ def upload(self, from_infos, to_infos, names=None):
             cb = Callback(name)
 
             try:
+                tmp_file = tmp_fname(to_info["path"])
```

Comment:
This is not needed for Azure, GS and S3, because they take care of that by themselves. Broken upload will not be shown in the bucket. I've noted it in https://github.com/iterative/dvc/issues/1674#issue-416183404 . Am I missing something?

Reply:
Oh no, I was the one who forgot to take into account that!